src/Tizen.NUI/src/public/Common/BaseHandle.cs

```diff
@@ -565,22 +559,6 @@ protected virtual void Dispose(DisposeTypes type)
                 Log.Error("NUI", $"[ERR] SwigCPtr is NULL, need to check! me:{me}");
             }
 
-            if (swigCPtrCopy.Handle != global::System.IntPtr.Zero)
-            {
-                swigCMemOwn = false;
-                var nativeSwigCPtr = swigCPtrCopy.Handle;
-
-                // Remove swigCPtrCopy first. Now HasBody() return false.
-                swigCPtrCopy = new global::System.Runtime.InteropServices.HandleRef(null, global::System.IntPtr.Zero);
-                Interop.BaseHandle.DeleteBaseHandle(nativeSwigCPtr);
```

Comment:
We should call  `Interop.BaseHandle.DeleteBaseHandle(swigCPtr.Handle);` somewhere. (Maybe after `swigCPtr = new global::System.Runtime.InteropServices.HandleRef(null, global::System.IntPtr.Zero);` finished, so `HasBody()` return false)

Without call DeleteBaseHandle API, It will make memory leak

Reply:
Most of classes (like `Animatable` and `Disposable`) implement there custom `ReleaseSwigCPtr` so it might not a problem.
But someone might not implement it.

I think the best position to implement `Interop.BaseHandle.DeleteBaseHandle(swigCPtr.Handle)` is `BaseHandle.ReleaseSwigCPtr(~~)`, which is not implemented anything now.

Reply:
Thank you very much.
I have changed following your review.